source/edge-server/connect.txt

```diff
@@ -170,58 +203,122 @@ modifications you can make to the :github:`Swift template app
+
+.. list-table::
+   :header-rows: 1
+
+   * - Element
+     - Description
+
+   * - ``<username>``
+     - The username of the Atlas user. For an 
+       :ref:`email/password user <email-password-authentication>`, this 
+       might look similar to ``first.last@example.com``.
+
+   * - ``<password>``
+     - The password of the Atlas user. This might look similar to 
+       ``myPassword123456``.
 
-The ``EDGE_SERVER_HOSTNAME`` is the public DNS of your Edge Server host. 
-For more information, refer to :ref:`edge-server-install-and-configure`.
+   * - ``<hostname>``
+     - The public DNS of your Edge Server host. While in local development, 
+       this might be ``localhost``. Otherwise, it might look similar to
+       ``ec2-13-58-70-88.us-east-2.compute.amazonaws.com``. For more 
+       information, refer to :ref:`edge-server-install-and-configure`.
 
-You must make port 27021 accessible to the client through port mapping or
-firewall rules.
+   * - ``<port>``
+     - The Edge Server Wire Protocol listen port. The default port is ``27021``.
+       For more details, refer to :ref:`edge-server-install-and-configure`.
+
+   * - ``authMechanism=PLAIN``
+     - The auth mechanism. Currently, the only supported value is ``PLAIN``.
+
+   * - ``appName``
+     - Optional string ``appName``. You may omit the value and/or key.
+
+   * - ``<auth-type>``
+     - The type of authentication provider. For example, ``local-userpass`` 
+       for email/password authentication. Currently, the only supported types
+       are ``anonymous`` or ``local-userpass``.
 
 .. example::
 
-   A wire protocol connection URI for an Edge Server hosted on Amazon EC2 
-   might look similar to:
+   A Wire Protocol connection URI for an Edge Server instance hosted on Amazon 
+   EC2 might look similar to:
 
    .. code:: console
 
-      mongodb://ec2-13-58-70-88.us-east-2.compute.amazonaws.com:27021
+      mongodb://example-user:my-password@ec2-13-58-70-88.us-east-2.compute.amazonaws.com:27021?authMechanism=PLAIN&appName=::local-userpass
+
+Supported Authentication Providers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Edge Server currently supports a subset of authentication providers when
+connecting over the Wire Protocol:
+
+- :ref:`anonymous-authentication`
+- :ref:`email-password-authentication`
+
+Supported MongoDB APIs
+~~~~~~~~~~~~~~~~~~~~~~
 
 Edge Server currently supports a subset of the MongoDB APIs. For information 
 about the supported APIs, refer to :ref:`edge-server-mongodb`.
+
+.. _edge-server-compass-connect:
+
+Connect to the Edge Server with Compass
+---------------------------------------
+
+You can connect to the Edge Server instance using 
+:ref:`MongoDB Compass <compass-index>`. Compass enables you to explore the 
+documents syncing to Edge Server instance, and make updates to those documents.
+
+[PLACEHOLDER - not sure exactly what URI to use to connect with Compass - 
```

Comment:
same URI as above for drivers. `mongodb://host:port` if anonymous, or `mongodb://user:pass@host:port` if userpass